File: ePrijevozSarajevo/ePrijevozSarajevo.Services/Database/Route.cs
namespace ePrijevozSarajevo.Services.Database
{
    public class Route
    {
        public int RouteId { get; set; }
        public Station? StartStation { get; set; } 
        public int StartStationId { get; set; }
        public Station? EndStation { get; set; }
        public int EndStationId { get; set; }
        public Vehicle? Vehicle { get; set; } 
        public int VehicleId { get; set; }
        public DateTime Departure { get; set; }
        public DateTime Arrival { get; set; }
    }
}

File: ePrijevozSarajevo/UI/eprijevoz_mobile/lib/models/route.dart
import 'package:json_annotation/json_annotation.dart';

part 'route.g.dart';

@JsonSerializable()
class Route {
  int? routeId;
  int? startStationId;
  int? endStationId;
  int? vehicleId;
  DateTime? arrival;
  DateTime? departure;
  //bool? active;
  //bool? activeOnHolidays;
  //bool? activeOnWeekend;

  Route(
      {this.routeId,
      this.startStationId,
      this.endStationId,
      this.vehicleId,
      this.arrival,
      this.departure});

  factory Route.fromJson(Map<String, dynamic> json) => _$RouteFromJson(json);

  Map<String, dynamic> toJson() => _$RouteToJson(this);
}
